### ISSUE TITLE
tabindex="100" prevents sequential keyboard navigation

### ISSUE THREAD

**Issue by IvanOliinychenko** [wenzhixin/bootstrap-show-password#52]
Using ```tabindex="100"``` prevents sequential keyboard navigation and [should be avoided](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).
I propose using ```tabindex="0"``` instead. 
This way navigation to the password toggle button could be done right after the password input which improves accessibility support.
